lib/kitchen/directions/bake_chapter_title/main.rb

```diff
@@ -0,0 +1,9 @@
+module Kitchen
+  module Directions
+    module BakeChapterTitle
+      def self.v1(book:)
+        V1.new.bake(book: book)
```

Comment:
You can write a simple test for this line:

```ruby
RSpec.describe Kitchen::Directions::BakeChapterTitle do
  it 'calls v1' do
    expect_any_instance_of(Kitchen::Directions::BakeChapterTitle::V1).to receive(:bake).with(book: "blah")
    described_class.v1(book: "blah")
  end
end
```

Later in this file we'd add `it 'calls v2'` etc

Reply:
I'm sure this is missing for the other existing directions specs too